src/components/Navbar/index.js

```diff
@@ -0,0 +1,31 @@
+import React, {Component} from 'react';
+import {withRouter, Link} from 'react-router-dom';
+import {ReactComponent as Logo} from '../../aab_logo.svg';
+import {Row, Col} from 'react-bootstrap';
+import './styles.scss';
+
+class Navbar extends Component {
+  render () {
+    return (
+      <Row>
+        <Col md={1}>
+          <Logo></Logo>
+        </Col>
+        <Col md={{offset : 5}} className="text">
+          ABOUT
+        </Col>
+        <Col className="text">
```

Comment:
Might be helpful to give it a name of like "navbarText" instead of just text... sorry this is so nitpicky 😅